recipes/miktex-portable/post-link.bat

```diff
@@ -0,0 +1,5 @@
+rem packages for nbconvert
```

Comment:
Not sure if you do any Windows development, @Korijn. If you do and I am missing the point, sorry. This is simply one way to do a comment in batch scripts on Windows.